Python/Watermark_to_a_video_with _a_given_font/README.md

```diff
@@ -0,0 +1,16 @@
+## watermark a video with a given font
+- This script built in Python, using this python script you can watermark a video. 
+- The OpenCV and Numpy Libraries are used to built this scripts.
+## Requirements
+- install OpenCv (use command:- "pip install opencv-python" to install OpenCv through Command Prompt)
+- install ffpyplayer (use command:- "pip install ffpyplayer" to install ffpyplayer through Command Prompt)
```

Comment:
i am getting error for ffpyplayer installation.  Can you please specify the version of this module you used

Reply:
It's 4.3.2, I think you have to upgrade your pip first, Because we didn't have to specify the version.
for more you can checkout this :- https://pypi.org/project/ffpyplayer/